book/src/external-provisioner.md

```diff
@@ -19,7 +19,7 @@ Latest stable release | Branch | Min CSI Version | Max CSI Version | Container I
 
 Latest stable release | Branch | Min CSI Version | Max CSI Version | Container Image | [Min K8s Version](kubernetes-compatibility.md#minimum-version) | [Max K8s Version](kubernetes-compatibility.md#maximum-version) | [Recommended K8s Version](kubernetes-compatibility.md#recommended-version) |
 --|--|--|--|--|--|--|--
-[external-provisioner v1.6.1](https://github.com/kubernetes-csi/external-provisioner/releases/tag/v1.6.1) | [release-1.6](https://github.com/kubernetes-csi/external-provisioner/tree/release-1.6) | [v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | - | k8s.gcr.io/sig-storage/csi-provisioner:v1.6.1 | v1.13 | - | v1.18
+[external-provisioner v1.6.1](https://github.com/kubernetes-csi/external-provisioner/releases/tag/v1.6.1) | [release-1.6](https://github.com/kubernetes-csi/external-provisioner/tree/release-1.6) | [v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | - | k8s.gcr.io/sig-storage/csi-provisioner:v1.6.1 | v1.13 | v1.21 | v1.18
```

Comment:
max k8s version should be updated for external-provisioner v1.5.0 and v1.4.0 as well.

Reply:
Valid point. Do you have a suggestion for the max K8s versions because I don't have. I mean, I can of course put v1.21 but I didn't tested myself this. And there are no good docs on external-provisioner side that actually can describe this.

Reply:
release 1.4 also uses beta CSINode: https://github.com/kubernetes-csi/external-provisioner/blob/release-1.4/pkg/controller/topology.go#L270